config/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "mt5": {
+        "symbol": "SOLUSD",
```

Comment:
Is this file supposed to define a symbol even though we now have one file per symbol?